src/App.js

```diff
@@ -1,25 +1,83 @@
-import logo from './logo.svg';
-import './App.css';
+import "./App.css";
+import { useState } from "react";
+import contactsData from './contacts.json'
+
 
 function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+
+const initialcontacts = contactsData.slice(0,5)
+const [contacts, setContacts] = useState(initialcontacts)
+
+// funtions
+
+function addRandomContact(){
+const contactsCopy = [...contacts]
```

Comment:
Niceee joob only we need a safe guard when we no have more contacts to add 

```suggestion
function addRandomContact(){
   if (contacts.length === contactsData.length) {
      console.log("No more contacts to add")
      return
    }
const contactsCopy = [...contacts]
```